targets.config.js

```diff
@@ -116,4 +116,30 @@ module.exports = {
         bundleFolder: 'electronBundle',
         mustExistFile: 'main.bundle.js',
     },
+    electronInsider: {
```

Comment:
I know you mentioned that this PR isn't meant to include some refactoring work, but what would you thoughts be on updating the current `electron` target here to `electronCanary` so that it's consistent with the new `electronInsider` and `electronProduction` targets?

Reply:
I am hoping we can come back and change the target names to either only be <channel> organized underneath product, or <product+channel>. My preference is the former, but in either case I tend towards leaving the current electron canary build alone until we settle on the others. This will avoid unintentional regressions in gruntfile.js and/or the current electron drop. It also feels like we might want an 'electronDev' target. @dbjorge would you prefer I update the electron target to electronCanary here?